client/src/components/game/lab2/LandingPage/mainInstructions.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Circle from '../components/circle';
+import './landingpage.css';
+
+const MainInstructions = () => {
+  return (
+    <div className='mainInstructionsContainer'>
+        {/*
+      <p className='instructionTitle'>
+        Main instructions for Color Clicker:
+      </p>
+        */}
+      <p className='mainInstructionList'>
+        You are about to play a game involving three colored circles, the same
+        size as this one:
+      </p>
+      <div className='center'>
+        <Circle color={'blue'} clickable={false}/>
+      </div>
+      <ul>
+        <li className='mainInstructionsItem'>
+          You will need to <strong>click the circle</strong> in the center of the screen.
```

Comment:
Maybe these instructions in list items could have bullet points? It might improve the readability